tests/runtests/bz652338-removed-proc-PID/runtest.sh

```diff
@@ -77,11 +74,11 @@ rlJournalStart
 
         rlAssertGrep "Skipping core dump" var-log-messages
 
-        rlRun "$CLI_LIST | grep package | grep abrt" 1 "abrtd did not crashed"
+        rlRun "abrt list --fmt={package} | grep abrt" 1 "abrtd did not crashed"
 
-        dirs="$($CLI_LIST | grep Directory | awk '{ print $2 }')"
+        dirs="$(abrt list --fmt={path})"
         for dir in $dirs; do
-            rlRun "$CLI_RM $dir" 0 "Dump dir removed ($dir)"
+            rlRun "abrt remove $dir" 0 "Dump dir removed ($dir)"
```

Comment:
Does this work without `-f`?

Reply:
True, it doesn’t.

Reply:
Wait, it does! I got it mixed up with `abrt remove` with no argument.

Reply:
Yes, it works.

Reply:
Agree to disagree.